packages/@okta/vuepress-theme-prose/components/Home.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <div class="homepage container-fluid">
     <div class="row">
-      <div class="col-xl-2 col-lg-2 d-none d-lg-block homepage--col-no-padding">
-        <HomeSidebar />
```

Comment:
nit: good to remove `HomeSidebar` import?

Reply:
yup, it's WIP yet